qiskit/pulse/macros.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Module for common pulse programming macros that construct pulse programs for common tasks."""
+
+from typing import Dict, List, Optional, Union
+
+from qiskit.pulse import channels
```

Comment:
In the future, you should try to move files in a single commit to keep the git blame. Then I would be able to see what was originally written by Soolu and what changes you made